src/components/widget/SelectionDropdown.js

```diff
@@ -129,9 +130,11 @@ export default class SelectionDropdown extends Component {
 
     switch (event.key) {
       case 'ArrowUp':
+        event.preventDefault();
         navigate(true);
         break;
       case 'ArrowDown':
+        event.preventDefault();
```

Comment:
`preventDefault` is already called [here](https://github.com/metasfresh/metasfresh-webui-frontend/blob/master/src/components/widget/SelectionDropdown.js#L153)